app/views/categories/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render 'shared/navbar', back_link: categories_path, title:"New Category" %>
+
+<%=form_for @category, html: {class:""} do |f| %>
+  <div class="input-field">
+    <%= f.label :name %>
+    <%= f.text_field :name %>
```

Comment:
- [ Required ] I think it would be better if you make this field a required field so as to prevent a user from adding a category with a blank name. I actually, succeeded in adding a category with an invalid name. Please see the screenshot below and rectify this.

## Example code
```ruby
  <%= f.text_field :name, {required: true} %>
```

## Screenshot
![Screenshot from 2023-08-09 02-10-12](https://github.com/Naledi-Dikgale/ror-budgetapp/assets/16392046/fe587386-89da-4e1f-9d66-c8e50c5b7bd3)